src/App.tsx

```diff
@@ -1,24 +1,202 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import classNames from 'classnames';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo, Errors, Filter } from './types/Todo';
+import { Form } from './components/Form';
+import { TodoList } from './components/Todolist';
+import { Footer } from './components/Footer';
+import { getTodos, postTodo, removeTodo } from './api/todos';
+import { ErrorNotification } from './components/ErrorNotification';
 
-const USER_ID = 0;
+const USER_ID = 6910;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<string>(Filter.ALL);
+  const [error, setError] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [todosBeingTransform, setTodosBeingTransform] = useState<number[]>([]);
+  const [isDeleted, setIsDeleted] = useState(false);
+  const showError = error.length > 0;
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(data => {
+        setTodos(data);
+        setError('');
+      })
+      .catch(() => {
+        setError(Errors.LOADING);
+      })
+      .finally(() => {
+        setTimeout(() => {
+          setError('');
+        }, 3000);
+      });
+  }, []);
+
+  const setErrorTimeout = () => {
+    setTimeout(() => {
+      setError('');
+    }, 3000);
+  };
+
+  const handleAddTodo = (todoData: Omit<Todo, 'id'>) => {
+    if (!todoData.title.trim()) {
+      setError(Errors.TITLE);
+      setErrorTimeout()
+      return;
+    }
+
+    setTempTodo({ ...todoData, id: 0 });
+
+    postTodo(todoData)
+      .then((newTodo: any) => (
```

Comment:
Type any is used for newTodo. Consider using a more specific type, like Todo, to avoid potential issues.